src/common/FilePreviewCommon/monacoSpecialLanguages.js

```diff
@@ -10,6 +10,8 @@ export async function registerAdditionalLanguages(monaco){
     registerAdditionalLanguage("txtExt", [".sln"], "txt", monaco)
     registerAdditionalLanguage("razorExt", [".razor"], "razor", monaco)
     registerAdditionalLanguage("logExt", [".log"], "log", monaco)
+    registerAdditionalLanguage("vbExt", [".vbs"], "vb", monaco)
+    registerAdditionalLanguage("iniExt", [".inf", ".gitconfig", ".gitattributes", ".editorconfig"], "ini", monaco)
```

Comment:
I believe .gitignore can be supported here as well

Reply:
No .gitignore uses a completly different syntax. And I added that in another PR

Reply:
right, I forgot about that PR. I even approved it already :P